modules/op-release-notes-1-16.adoc

```diff
@@ -0,0 +1,594 @@
++
+--
+* `loki_stack_name`: The name of the `LokiStack` CR, typically `logging-loki`.
+* `loki_stack_namespace`: The name of the namespace where LokiStack is deployed, typically `openshift-logging`.
+--
++
+.Example of configuring LokiStack log forwarding in the `TektonResult` CR
+[source,yaml]
+----
+apiVersion: operator.tekton.dev/v1
+kind: TektonResult
+metadata:
+  name: result
+spec:
+  targetNamespace: tekton-pipelines
+# ...
+  loki_stack_name: logging-loki
+  loki_stack_namespace: openshift-logging
+----
+
+[id="breaking-changes-1-16_{context}"]
+== Breaking changes
+
+* With this update, the metric name for the `EventListener` object for pipelines triggers that counts received events is changed from `eventlistener_event_count` to `eventlistener_event_received_count`.
+
+[id="known-issues-1-16_{context}"]
+== Known issues
+
+* The `jib-maven` `ClusterTask` does not work if you are using {OCP} version 4.16 and later.
+
+[id="fixed-issues-1-16_{context}"]
+== Fixed issues
+
+* Before this update, when you uninstalled {tekton-hub} by deleting the `TektonHub` CR, the pod of the `hub-db-migration` job was not deleted. With this update, uninstalling {tekton-hub} deletes the pod.
+
+* Before this update, when you used {tekton-results} to store pod logs from pipelines and tasks, the operation to store the logs sometimes failed. The logs would include the `UpdateLog` action failing with the `canceled context` error. With this update, the operation completes correctly. 
+
+* Before this update, when you passed a parameter value to a pipeline or task and this value included more than one variable, and they use both full and short reference formats, for example, `$(tasks.task-name.results.variable1) + $(variable2)`, {pipelines-shortname} did not interpret the value correctly; the pipeline run or task run could stop execution and the Pipelines controller could crash. With this update, {pipelines-shortname} interprets the value correctly and the pipeline run or task run completes.
+
+* Before this update, {tekton-chains} failed to generate correct attestations when a task run included multiple tasks with the same name. For instance, when using a matrix of tasks, the attestation was generated for the first image. With this update, {tekton-chains} generates attestations for all tasks within the task run, ensuring complete coverage.
+
+* Before this update, when you used the `skopeo-copy` task defined in the {pipelines-shortname} installation namespace and set its `VERBOSE` parameter to `false`, the task failed. With this update, the task completes normally.
+
+* Before this update, if you installed {pipelines-shortname} on a cluster with a high number of namespaces, the installation could take a significant time, because the Operator created RBAC resources in every namespace in a sequence. With this update, the Operator creates the RBAC resources concurrently, resulting in a faster installation.
```

Comment:
is this still true @jkhelil :thinking: 
asking because you have reverted the changes isn't it ?

Reply:
Yes, this was reverted, so we should remove it.
@eromanova97 
@savitaashture 
@mramendi